daemon/cmd/local_node_sync.go

```diff
@@ -142,15 +175,21 @@ func (ini *localNodeSynchronizer) initFromK8s(ctx context.Context, node *node.Lo
 	//   - IPsec key (set by IPsec)
 	//   - alloc CIDRs (depends on IPAM mode; restored from Node or CiliumNode)
 	node.Name = parsedNode.Name
+	node.Labels = parsedNode.Labels
+	node.Annotations = parsedNode.Annotations
+	node.Cluster = parsedNode.Cluster
```

Comment:
I guess these changes, as well as the one at line 153 below, are not intentional, and should be reverted. They are the reason why a couple of unit tests are failing.